packages/fx-core/src/component/driver/prerequisite/installDriver.ts

```diff
@@ -30,16 +30,19 @@ import { DotnetInstallationUserError } from "./error/dotnetInstallationUserError
 import { FuncInstallationUserError } from "./error/funcInstallationUserError";
 import { InstallToolArgs } from "./interfaces/InstallToolArgs";
 import { InvalidActionInputError } from "../../../error/common";
+import { addStartAndEndTelemetry } from "../middleware/addStartAndEndTelemetry";
+import { hooks } from "@feathersjs/hooks/lib";
 
 const ACTION_NAME = "prerequisite/install";
-const outputName = Object.freeze({
-  SSL_CRT_FILE: "SSL_CRT_FILE",
-  SSL_KEY_FILE: "SSL_KEY_FILE",
-  FUNC_PATH: "FUNC_PATH",
-  DOTNET_PATH: "DOTNET_PATH",
-});
 const helpLink = "https://aka.ms/teamsfx-actions/prerequisite-install";
 
+const outputKeys = {
+  sslCertFile: "sslCertFile",
```

Comment:
Although they are all optional, will we check mis-match? E.g., do not install cert but output has sslCertFile.

Reply:
I prefer not. We can let user know in yaml schema that `sslCertFile` only takes effect when `devCert` is specified.